src/config_file.c

```diff
@@ -130,6 +131,24 @@ gboolean load_config(const gchar *filename, RaucConfig **config, GError **error)
 		c->system_bb_statename = g_key_file_get_string(key_file, "system", "barebox-statename", NULL);
 	}
 
+	c->max_bundle_download_size = g_key_file_get_uint64(key_file, "system", "max-bundle-download-size", &ierror);
```

Comment:
using `max-bundle-download-size` could add irritation about the actual unit used here MiB? KiB? Bytes?

I would suggest naming it `max-bundle-download-size-bytes` as I fear introducing units will add too much overhead.

Reply:
Puh... I understand your point, but I think appending -bytes to an already longish name is not nice either. As you said, I also guess that this config option is rarely used, so people should RTFM to be sure about the unit. Obmitting the -bytes suffix also allows to add an unit parsing later if it ever will become important...
So, I'll update my PR without this change first, but if it is a no-go for you to get this in, I'll change it...

BTW: What are the reasons for the considerations to mark this obsolete? I think this is a really cool feature, because there is no need to temporary store the whole file locally. And while I see the pro's of casync, I'm not sure that everybody wants to use it. One problem I see, is that distribution of a single file to customers web server for deployment is simpler than merging files of a chuck store.

Reply:
Ok, I understand your concerns regarding the longish name, but than at least your have to WTFM :) Cause atm it does not indicate which unit you have to use or if a value of "8MiB" is a valid or invalid one.
Adding something like "Specify maximum bundle download size in number of bytes, e.g. ``max-bundle-download-size =8*1024*1024``" should be sufficient.

Regarding the network mode: The real 'network mode ' was invented in the early days of RAUC but did not have too many users until now (meaning we know of a single one!). Since the initial patches this was actually basically tested but did not get many of the changes and features the normal bundle mode has.

When I talk about network mode, this also has nothing to do with bundles. It means having a single manifest downloaded from the web server and downloading the individual images described in it separately. It does not cover downloading a normal bundle over network. Is that actually how you use it? Using it for downloading normal bundles is not intended to work at the moment.

Reply:
I agree: leaving the door open to add a unit parser later is a good thing.
@mhei: As you are presumably going to prepare an updated PR with respect to the documentation would you mind renaming the macro `DEFAULT_MAX_DOWNLOAD_BUNDLE_SIZE` to `DEFAULT_MAX_BUNDLE_DOWNLOAD_SIZE` to have it consistent with the variable's name? (This is no must, but just a gentle request).